libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -905,7 +908,7 @@ open class SalesforceSDKManager protected constructor(
             account = account,
             frontActivity = frontActivity,
             showLoginPage = showLoginPage,
-            reason = LogoutReason.UNKNOWN
+            reason = UNKNOWN
```

Comment:
Slightly unrelated, but I did notice in the diff for the previous pull request I forgot this static import and that's always a nice way to reduce visual clutter.  I could undo this if we wish.